scripts/effect-definitions.js

```diff
@@ -84,759 +91,442 @@ export default class EffectDefinitions {
       this._encumbered,
       this._flanked,
       this._flanking,
+      this._favouredEnemy,
+      this._greaterFavouredEnemy,
       this._greatWeaponMaster,
       this._heavilyEncumbered,
       this._rage,
+      this._ragePathOfTheTotemWarrior,
```

Comment:
This is already handled dynamically with the Rage effect. When `isDynamic` is set to true, it'll use the `dynamic-effects-adder.js` to see what additional effects it should add. I specifically handle Path of the Totem Warrior barbarians in the rage case in that class. Recommend removing this.

Reply:
Removed rage path of the totme warrior